gallery/src/components/NxCollapsibleItems/NxCollapsibleItemsPage.tsx

```diff
@@ -0,0 +1,209 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import { NxTable, NxP, NxCode, NxTile, NxH3 } from '@sonatype/react-shared-components';
+
+import { GalleryDescriptionTile, GalleryExampleTile } from '../../gallery-components/GalleryTiles';
+
+import NxCollapsibleItemsExample from './NxCollapsibleItemsExample';
+import NxCollapsibleItemsTooltipExample from './NxCollapsibleItemsTooltipExample';
+import NxCollapsibleItemsExtras from './NxCollapsibleItemsExtrasExample';
+import NxCollapsibleItemsClickable from './NxCollapsibleItemsClickableExample';
+import NxCollapsibleItemsClickableSidebar from './NxCollapsibleItemsClickableSidebarExample';
+import NxCollapsibleItemsCheckbox from './NxCollapsibleItemsCheckboxExample';
+import NxCollapsibleItemsEmpty from './NxCollapsibleItemsEmptyExample';
+import NxCollapsibleItemsDisabled from './NxCollapsibleItemsDisabledExample';
+
+const NxCollapsibleItemsCode = require('./NxCollapsibleItemsExample?raw'),
+    NxCollapsibleItemsTooltipCode = require('./NxCollapsibleItemsTooltipExample?raw'),
+    NxCollapsibleItemsExtrasCode = require('./NxCollapsibleItemsExtrasExample?raw'),
+    NxCollapsibleItemsClickableCode = require('./NxCollapsibleItemsClickableExample?raw'),
+    NxCollapsibleItemsClickableSidebarCode = require('./NxCollapsibleItemsClickableSidebarExample?raw'),
+    NxCollapsibleItemsCheckboxCode = require('./NxCollapsibleItemsCheckboxExample?raw'),
+    NxCollapsibleItemsEmptyCode = require('./NxCollapsibleItemsEmptyExample?raw'),
+    NxCollapsibleItemsDisabledCode = require('./NxCollapsibleItemsDisabledExample?raw');
+
+const NxCollapsibleItemsPage = () =>
+  <>
+    <GalleryDescriptionTile>
+      <NxP>
+        A set of default styles and basic React for an expanding tree view.
```

Comment:
This description needs to be updated since it's no longer a tree.

Reply:
> This description needs to be updated since it's no longer a tree.

Fixed descriptions:
a431a5623b68c916560e31a0431e53edb5a634ac
a431a5623b68c916560e31a0431e53edb5a634ac